code/modules/power/apc.dm

```diff
@@ -75,6 +75,7 @@
 	var/locked = 1
 	var/coverlocked = 1
 	var/aidisabled = 0
+	var/tdir = null
```

Comment:
This isn't very descriptive, but if I had to guess, it means terminal direction?